src/geo/ui/infowindow.js

```diff
@@ -185,13 +185,13 @@ cdb.geo.ui.InfowindowModel = Backbone.Model.extend({
     options = options || {};
     var render_fields = [];
     for(var j = 0; j < fields.length; ++j) {
-      var f = fields[j];
-      var value = String(attributes[f.name]);
```

Comment:
no idea, but If I did, i did it because of something. My guess is: as we used this code in cartodb.js and the editor and we wanted to render null values as "null" in the editor I did that hack.

I think is ok to remove it